src/plugins/visualizations/public/vis_types/types.ts

```diff
@@ -48,6 +54,7 @@ export interface VisType<TVisParams = unknown> {
   readonly icon?: IconType;
   readonly image?: string;
   readonly stage: 'experimental' | 'beta' | 'production';
+  readonly group?: VisGroups;
```

Comment:
Since we always set a default value here when instantiating the VisType, we can remove the `?` since this will always be present.